CHANGES/815.feature

```diff
@@ -0,0 +1,2 @@
+Added support for automatically saving pull-through content to a repository.
+Requires pulpcore 3.74 or later.
```

Comment:
Where's the lower bound bump?  pyproject.toml needs updating.

Reply:
No, we don't. If you have >=3.74 then you get the new behavior, else you don't.

Reply:
Ok, that wasn't clear.  Thanks